lib/embed/viewer/was_seed.rb

```diff
@@ -19,6 +33,13 @@ def self.supported_types
         [:"webarchive-seed"]
       end
       
+      def thumbs_list
+        Embed::WasSeedThumbs.new(@purl_object.druid).get_thumbs_list
```

Comment:
It seems like we are going to need this on initialization, can we go ahead and do so on creation?